requirements-dev.txt

```diff
@@ -1,7 +1,7 @@
 # The Ubuntu GUI exits immediately without error in 6.11.0.
-cx_Freeze==6.10
-cx_Oracle
-sphinx
+cx_freeze==6.10
+cx-oracle==8.3.0
```

Comment:
This looks wrong. An underscore has been replaced by a dash. Will check why.

Reply:
[Package names are case-insensitive](https://peps.python.org/pep-0426/#name). I personally prefer the canonical names

Reply:
I have no strong personal opinion on this, please advise if you want me to change.

Reply:
I'd appreciate if you restore the names (and fix the existing `CherryPy` and `PyMySQL` names), but that won't hold up merging this PR.

There appears to have been an issue with the support@arelle.org email address that I believe you sent your signed CLA to (sorry for the inconvenience). Can you send a copy to Austin.Matherne@Workiva.com so we can get this merged?

Reply:
No problem! I had a look at the pypi page of each dependency and used the name state there. For `OpenPyXL` I was uncertain so I had a look at the lib's documentation page and it looks like they are using the `OpenPyXL` name there.